google/cloud/storage/internal/curl_client.cc

```diff
@@ -262,6 +264,12 @@ std::pair<Status, ObjectMetadata> CurlClient::InsertObjectMedia(
       request.GetOption<Fields>().value().empty()) {
     return InsertObjectMediaXml(request);
   }
+
+  // If the application has set an explicit
```

Comment:
Looks like the thought in your comment was left unfinished :P

Reply:
Ooops, thanks for noticing that. Fixed.